packages/tag/src/components/Tag.js

```diff
@@ -16,7 +16,13 @@ const Tag = props => {
       )}
       {text || children}
       {onRemove && (
-        <div className="close-icon" onClick={() => onRemove(value)}>
+        <div
+          role="button"
+          tabIndex="0"
+          className="close-icon"
+          onClick={() => onRemove(value)}
+          onKeyDown={() => onRemove(value)}
```

Comment:
It's kinda weird allow any key pressing to call `onRemove`, doesn't it? We could restrict some keys (`enter` and `space`) to call the action...

Reply:
Yep, even `tab` is triggering `onRemove`. So a user can't navigate with keyboard through the tags without removing them.

![2019-03-04 16 09 23](https://user-images.githubusercontent.com/1459283/53756620-53007200-3e98-11e9-9ba6-8567b3e6c04c.gif)